pkg/apis/serving/v1alpha1/revision_lifecycle_test.go

```diff
@@ -574,6 +574,38 @@ func TestRevisionGetLastPinned(t *testing.T) {
 	}
 }
 
+func TestRevisionIsReachable(t *testing.T) {
+	tests := []struct {
+		name   string
+		labels map[string]string
+		want   bool
+	}{{
+		name:   "has route annotation",
+		labels: map[string]string{serving.RouteLabelKey: "the-route"},
+		want:   true,
+	}, {
+		name:   "empty route annotation",
+		labels: map[string]string{serving.RouteLabelKey: ""},
+		want:   false,
+	}, {
+		name:   "no route annotation",
+		labels: nil,
+		want:   false,
+	}}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			rev := Revision{ObjectMeta: metav1.ObjectMeta{Labels: tt.labels}}
+
+			got := rev.IsReachable()
+
+			if got != tt.want {
```

Comment:
nit:

```suggestion
			if got := rev.IsReachable(); got != tt.want {
```

Reply:
Since this is a test, i'd prefer to keep the arrange/act/assert pattern for clarity.

Reply:
Why? Above we actually do this. So let's be consistent and apoly the pattern throughout